packages/react-devtools-shared/src/bridge.js

```diff
@@ -181,8 +181,7 @@ export type BackendEvents = {
   fastRefreshScheduled: [],
   getSavedPreferences: [],
   inspectedElement: [InspectedElementPayload],
-  isBackendStorageAPISupported: [boolean],
-  isSynchronousXHRSupported: [boolean],
+  isReloadAndProfileSupported: [boolean],
```

Comment:
Can it be `backendSupportsReloadAndProfile` or `isReloadAndProfileSupportedByBackend`?

The final version is up to you, I just want it to explicitly mention that this is about backend.